base/setup/lib/utils/partlist.c

```diff
@@ -2849,6 +2901,7 @@ CreatePartition(
 {
     ERROR_NUMBER Error;
     BOOLEAN isContainer = IsContainerPartition((UCHAR)PartitionInfo);
+    PDISKENTRY DiskEntry;
     PCSTR mainType = "Primary";
```

Comment:
Any reason not to use PCWSTR?

Reply:
These are just for having something to display in the DPRINT1's below (see for example:
`DPRINT1("CreatePartition(%s, %I64u bytes)\n", mainType, SizeBytes);` )
so I just keeping things simple here.